jsapp/js/components/transferProjects.module.scss

```diff
@@ -0,0 +1,65 @@
+@use 'scss/sizes';
+@use '~kobo-common/src/styles/colors';
+
+.root {
+  background: colors.$kobo-white;
+  padding: sizes.$x20;
+  margin: sizes.$x24 sizes.$x40;
+  border: 1px solid colors.$kobo-gray-92;
+  border-radius: sizes.$x5;
```

Comment:
I wonder if we should make a SCSS mixin out of this… Similarly to `floatingRoundedBox`. Maybe `inlineRoundedBox`? I imagine we will use this more often? Probably good to ask Tessa :)